### ISSUE TITLE
As a Project Coordinator, I want to see project revisions, so that I can understand what was changed

### ISSUE THREAD

**Issue by matthieu-foucault** [bcgov/cas-cif#240]
#### Description:

Brief description of the work involved, including any user research context or key decisions.

#### Acceptance Criteria:

Given
When
Then

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

#### Notes:

-

**Comment by Sriram-pc** [comment]
![CIF Workflow Co-Design - Edit Project - Revision tab.jpg](https://images.zenhubusercontent.com/5c2e4cfdab4f3464fb8df8d1/92f72ccd-8603-4fe9-b8e4-7455cb62fb67)

![CIF Workflow Co-Design - Edit Project - Revision tab - Details.jpg](https://images.zenhubusercontent.com/5c2e4cfdab4f3464fb8df8d1/ecfbb375-30dd-4509-a905-0a27d3a07ed8)

**Comment by matthieu-foucault** [comment]
This needs to be split in two issues

**Comment by LindsayMacfarlane** [comment]
@dleard - is this ticket complete with your most recent diff work? Or is this something different?

**Comment by dleard** [comment]
This is different, the work I did shows the diff when editing a project. This cards is about being able to view diffs between one revision to another

**Comment by LindsayMacfarlane** [comment]
Do we still need this ticket or has it been replaced with the existing tickets that have been created for revisions and amendments?

**Comment by LindsayMacfarlane** [comment]
Hi @pbastia and @nanyangpro - is this ticket still required or has it been replaced by any of the new amendment/revision user stories? If it is still required, I'd like to change it to be captured under amendments/revisions EPIC - thoughts?

**Comment by nanyangpro** [comment]
I think #920 and #994 cover this ticket, @pbastia?

**Comment by pbastia** [comment]
@nanyangpro not quite, this feature is about showing the diff between 2 arbitrary versions, as an analytics tool. It would be in addition to the 2 cards you linked

**Comment by nanyangpro** [comment]
Gotcha, thanks for clarifying @pbastia. Since this ticket was created in Jan, and wasn't specifically brought up by the CIF team during my discovering for amendments/revisions, I'm wondering if you still see the need for keeping it?

**Comment by Dianadec** [comment]
@LindsayMacfarlane  flagging this card for your review, as per Nan's comment above.

**Comment by LindsayMacfarlane** [comment]
@Dianadec - I feel we can icebox this ticket for now. This was a very early ticket in our development and would be better to validate with CIF if it is required. The functionality currently being built will most likely meet CIFs immediate needs and this may be more of a future need (if at all).

**Comment by LindsayMacfarlane** [comment]
I'm going to remove this ticket from the amendments EPIC as it is not a priority to get this feature working and in the hands of CIF.
